lib/Ed25519KeyPair.js

```diff
@@ -28,20 +28,25 @@ class Ed25519KeyPair extends LDKeyPair {
    * > const EDKey = new Ed25519KeyPair(options);
    * > EDKey
    * Ed25519KeyPair { ...
-   * @param {KeyPairOptions} options - Base58 keys plus
-   * other options most follow
+   * @param {KeyPairOptions} options - Options hashmap.
    * [KeyPairOptions]{@link ./index.md#KeyPairOptions}.
    * @param {string} options.publicKeyBase58 - Base58 encoded Public Key
-   * unencoded is 32-bytes.
-   * @param {string} options.privateKeyBase58 - Base58 Private Key
-   * unencoded is 64-bytes.
+   *   (unencoded is 32-bytes).
+   * @param {string} [options.privateKeyBase58] - Base58 Private Key
+   *   (unencoded is 64-bytes).
    */
   /* eslint-enable */
   constructor(options = {}) {
     super(options);
     this.type = 'Ed25519VerificationKey2018';
-    this.privateKeyBase58 = options.privateKeyBase58;
     this.publicKeyBase58 = options.publicKeyBase58;
+    if(!this.publicKeyBase58) {
+      throw TypeError('The "publicKeyBase58" property is required.');
```

Comment:
```suggestion
      throw new TypeError('The "publicKeyBase58" property is required.');
```

Please add a test that would have caught this.

Reply:
@mattcollier - 1) thank you for the catch! 
2) Ok, so, that's puzzling. I DO have a [test for that line](https://github.com/digitalbazaar/crypto-ld/pull/45/files#diff-40d036c1031e5d239c9f4d7ec8e83749R36). I wonder why it didn't catch it??

Reply:
@mattcollier yeah, I just double-checked - unfortunately it's syntax-legal to say `throw TypeError` and forget the new. So I think eslint is the only way to catch that error.

Reply:
Fixed.